third_party/amd/lib/TritonAMDGPUToLLVM/LoadStoreOpToLLVM.cpp

```diff
@@ -588,12 +588,17 @@ struct AsyncCopyGlobalToLocalOpConversion
     // size
     for (int i = 0; i < shmemAddrs.size(); i++) {
       auto srcIdx = i * maxVec;
-      auto srcPtr = srcElems[srcIdx];
+      Value srcPtr = srcElems[srcIdx];
 
       if (maskElems.empty()) {
         rewriter.create<ROCDL::GlobalLoadLDSOp>(
-            loc, srcPtr, shmemAddrs[i], vecBytesVal, /*offset=*/b.i32_val(0),
-            cacheModifiers);
+            loc,
+            /*globalPtr*/ srcPtr,
+            /*ldsPtr*/ shmemAddrs[i],
+            /*size*/ vecBytesVal,
+            /*offset=*/b.i32_val(0),
+            /*aux*/ cacheModifiers, /*alias_scopes*/ nullptr,
+            /*noalias_scopes*/ nullptr, /*tbaa*/ nullptr);
```

Comment:
Nit: please use consistent style `=*/` and concatenate them into fewer lines. :)

Reply:
i can fix up the `=*/` of course but I didn't choose the new lines - that's `clang-format`

Reply:
okay clang-format was happy with two per line